apm-sniffer/apm-sdk-plugin/mysql-6.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/jdbc/mysql/v6/define/StatementInstrumentation.java

```diff
@@ -79,6 +77,11 @@
     }
 
     @Override protected ClassMatch enhanceClass() {
-        return byMultiClassMatch(STATEMENT_CLASS_NAME, MYSQL6_STATEMENT_CLASS_NAME);
+        return byName(MYSQL6_STATEMENT_CLASS_NAME);
+    }
+
+    @Override
+    protected String[] witnessClasses() {
```

Comment:
I advise that this method should remove. because this method had implemented in its parent class